FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -1,7 +1,108 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MemberInfoCard.module.scss';
+import Card from 'react-bootstrap/Card';
+import { contactIcons } from '@icons';
+import { Link, FilledImage } from '@basics';
+import { Body1, Body2, Caption } from '@components/basics';
+import cn from 'classnames';
 
-const MemberInfoCard: FunctionComponent = () => <Container></Container>;
+export interface MemberInfoCardProps {
+  name: string;
+  imgSrc: string;
+  title: string;
+  quote: string | { number: number; length: string };
+  website?: string;
+  email?: string;
+  linkedIn?: string;
+}
+
+const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
+  name,
+  imgSrc,
+  title,
+  quote,
+  website,
+  email,
+  linkedIn,
+}) => {
+  //establishing a first name for use in text bodeis
+  var nameArray: string[] = name.split(' ');
```

Comment:
Prefer "const" over "var". If you need to change the value, then use "let" instead of "var"